sessions/2019/january/Session_22.md

```diff
@@ -0,0 +1,135 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 22 (15/01/2019)
+
+- Java Memory Handling
+- Fixing Logger implmeentation
+- Sprint workflow in GitHub project for estimate effort
+- Lint checker
+- Microservices resources
+- Git advices
+
+### Notes
+
+#### Java memory
+
+![Java memory](https://cdncontribute.geeksforgeeks.org/wp-content/uploads/jvm-3.jpg "Java memory")
+
+Java uses their JVM to distribute its own memory use.
+
+In `Heap` is the zone where variables are. Pointers are in `JVM Language Stacks`.
```

Comment:
"variables" is a bit vague. It's the *values of the data*, I.e.` "String foo = "bar";` -> here the `foo` reference exists in the stack, but `"bar"` exists in the heap. Pointers aren't really a thing in Java - prefer to use 'references' instead.